crypto/ts/ts_rsp_verify.c

```diff
@@ -272,6 +309,41 @@ static int ts_find_cert(STACK_OF(ESS_CERT_ID) *cert_ids, X509 *cert)
     return -1;
 }
 
+/* Returns < 0 if certificate is not found, certificate index otherwise. */
+static int ts_find_cert_v2(STACK_OF(ESS_CERT_ID_V2) *cert_ids, X509 *cert)
+{
+    int i;
+    unsigned char cert_digest[EVP_MAX_MD_SIZE];
+    unsigned int len;
+
+    if (cert_ids == NULL || cert == NULL)
```

Comment:
are NULL valid values, or do they mean programming error?  if programming error, then remove this test.